cumulusci/core/github.py

```diff
@@ -444,6 +453,12 @@ def format_github3_exception(exc: Union[ResponseError, TransportError]) -> str:
         sso_error_msg = check_github_sso_auth(exc)
         user_warning = scope_error_msg + sso_error_msg
 
+    if isinstance(exc, ConnectionError):
+        if "self signed certificate" in str(exc.exception):
```

Comment:
💭 If I'm understanding correctly, this _should_ be platform independent. If I'm not, we may see some other self-signed error messages that won't match. I think that's a reasonable trade-off.